Admin/BaseFieldDescription.php

```diff
@@ -415,20 +416,6 @@ public function getMappingType()
     }
 
     /**
-     * Camelize a string.
-     *
-     * @static
-     *
-     * @param string $property
-     *
-     * @return string
-     */
-    public static function camelize($property)
-    {
-        return Container::camelize($property);
-    }
```

Comment:
Why not ? It's only a matter of adding a deprecation warning instead of removing the method, which would make the upgrade easier and the branches less different.